docker-run.sh

```diff
@@ -29,7 +29,7 @@ EOF
 \$dolibarr_main_document_root_alt='/var/www/html/custom';
 \$dolibarr_main_data_root='/var/www/documents';
 \$dolibarr_main_db_host='${DOLI_DB_HOST}';
-\$dolibarr_main_db_port='${DOLI_DB_HOST_PORT}';
+\$dolibarr_main_db_port='3306';
```

Comment:
Better not hardcode this, DB is not part of the image and so you cannot be sure the implementation the client did on his infrastructure (changing standard port is a common way to secure your infra)...
It also could be a problem for a version compatible with other database engines such as Postgres.

Reply:
I agree.

@MkLHX your first PR #41 is good. No need to revert it.

Reply:
@mathieupotier ok i udenstand i just revert my edits.
the commit before my PR have this port harcoded.
https://github.com/tuxgasy/docker-dolibarr/blob/737234da339b16a40db762654147cda37c617617/docker-run.sh